travvy/src/store/store.js

```diff
@@ -6,19 +6,26 @@ Vue.use(Vuex)
 export default new Vuex.Store({
     strict: true, 
     state: {
-        // Creates the recommended attractions variable where an array of recommended attractions will be stored
-        recommendedAttractions: null
+        // Creates the recommended attractions and city variable where an array of recommended attractions will be stored
+        recommendedAttractions: null, 
+        city: null
     },
     mutations: {
         // If user selects new locations, recommended attractions change
         setRecommendedAttractions(state,recommendedAttractions){
             state.recommendedAttractions = recommendedAttractions
+        }, 
+        setCity(state,city){
```

Comment:
had to add a setter for city. If the recommended attractions is empty, as a result it is not possible to extract the city from the recommended attractions and need to save it separately based on the user input